code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -1028,6 +1029,24 @@
 	..()
 	to_chat(user, "<span class='hierophant_warning'>The[beacon ? " beacon is not currently":"re is a beacon"] attached.</span>")
 
+/obj/item/hierophant_club/suicide_act(mob/living/user)
+	user.say("Xverwpsgexmrk...")
```

Comment:
I think you should not have any lines in the hierophant's rotated text that aren't from it.
like, really, it's a big machine dealy, at least have the club itself speak instead.